stock_picking_report_valued/models/stock_move_line.py

```diff
@@ -49,11 +50,16 @@ def _compute_sale_order_line_fields(self):
         """
         for line in self:
             sale_line = line.sale_line
+            sale_line_uom = sale_line.product_uom
             price_unit = (
                 sale_line.price_subtotal / sale_line.product_uom_qty
                 if sale_line.product_uom_qty
                 else sale_line.price_reduce
             )
+            if sale_line_uom.uom_type == "bigger":
```

Comment:
Please use `uom.uom` `convert` method for changing the quantity in the previous expression.

Reply:
I used `_compute_price` method as I found no `convert` method